### ISSUE TITLE
Is it possible to specify a ARDL without intercept using this package?

### ISSUE THREAD

**Issue by mabuimo** [Natsiopoulos/ARDL#13]
ardl_fit <- ardl(target~ -1 + cpi+ oil + ind_prod, data = france_tbl, order = c(1,1,2,1))
I have tried using -1 in the formula but it doesn't work.

**Comment by Natsiopoulos** [comment]
Yes, it is possible to specify an ARDL model without an intercept.
A bug doesn't allow `-1` to be right after `~`.
Thank you for reminding me of this issue, I will fix it.
For now, just place `-1` in another place, between the other variables, or at the end (e.g. ardl(target~ cpi+ oil + ind_prod -1, data = france_tbl, order = c(1,1,2,1)))
